ast.go

```diff
@@ -1218,7 +1275,7 @@ func (*TableConstraintCheck) iTableConstraint()      {}
 // TableConstraintPrimaryKey is a PRIMARY KEY constraint for table definition.
 type TableConstraintPrimaryKey struct {
 	Name    Identifier
-	Columns ColumnList
+	Columns IndexedColumnList
```

Comment:
So a Table Constraint Primary Key is now a list of indexed columns as described in previous comment.